### PR TITLE
Consume AAO config in client for local dev

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Consume AAO config in client for local dev

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

On local dev

### How will this change be monitored?

For features that are critical or could fail silently please describe the monitoring/alerting being added.

### CODE REVIEW

File: src/pages/audio-rewards-page/store/sagas.ts [hunk 1]
```diff
@@ -96,12 +100,21 @@ function* claimChallengeRewardAsync(
...
   )
+  if (ENVIRONMENT === 'development') {
+    const oracleEthAddresses = (REACT_APP_ORACLE_ETH_ADDRESSES || '').split(',')
+    if (oracleEthAddresses.length > 0) {
+      oracleEthAddress = oracleEthAddresses[0]
+    }
+    if (REACT_APP_AAO_ENDPOINT) {
+      AAOEndpoint = REACT_APP_AAO_ENDPOINT
+    }
+  }
```
Comment (piazzatron) [inline src/pages/audio-rewards-page/store/sagas.ts:117]: Could probably pull this whole bit into a function

**Review by piazzatron — APPROVED** [review]
Looks great 🚀

File: src/pages/audio-rewards-page/store/sagas.ts (hunk 1 above)
Comment (sddioulde) [reply src/pages/audio-rewards-page/store/sagas.ts:117]: let's do it

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-aao-local-dev-config

**Pull request merged**